sql-db/sql-app/src/test/java/io/quarkus/ts/sqldb/sqlapp/DevModeMariadbDevServicesUserExperienceIT.java

```diff
@@ -17,7 +17,7 @@
 public class DevModeMariadbDevServicesUserExperienceIT {
 
     private static final String MARIA_DB_VERSION = "5.5.49";
```

Comment:
This is an ancient tag, last pushed into 6 years ago - we should definitely bump it to 10.8 (I guess), and drop the micro release version